config/seata-config-core/src/main/java/io/seata/config/ConfigurationFactory.java

```diff
@@ -111,27 +110,45 @@ private static Configuration buildConfiguration() {
         } catch (Exception e) {
             throw e;
         }
+        Configuration extConfiguration = null;
+        Configuration configuration;
         if (ConfigType.File == configType) {
-            String pathDataId = String.join(ConfigurationKeys.FILE_CONFIG_SPLIT_CHAR, ConfigurationKeys.FILE_ROOT_CONFIG, FILE_TYPE, NAME_KEY);
+            String pathDataId = String.join(ConfigurationKeys.FILE_CONFIG_SPLIT_CHAR,
+                ConfigurationKeys.FILE_ROOT_CONFIG, FILE_TYPE, NAME_KEY);
             String name = CURRENT_FILE_INSTANCE.getConfig(pathDataId);
-            Configuration configuration = new FileConfiguration(name);
-            Configuration extConfiguration = null;
+            configuration = new FileConfiguration(name);
             try {
                 extConfiguration = EnhancedServiceLoader.load(ExtConfigurationProvider.class).provide(configuration);
```

Comment:
ExtConfigurationProvider will have multiple implementation classes, here you need to specify which extension classes to load.

Reply:
ok